src/Pelagos/Bundle/AppBundle/Resources/views/Search/default.html.twig

```diff
@@ -3,19 +3,44 @@
 {% block head %}
     {{ add_css (
         [
-            '//cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css'
+            '//netdna.bootstrapcdn.com/bootstrap/3.0.3/css/bootstrap.min.css'
         ]
     ) }}
     {% stylesheets
         '@PelagosAppBundle/Resources/public/css/search.css' %}
         {{ add_css(asset_url) }}
     {% endstylesheets %}
+
+    {{ add_js (
+        [
+            '//code.jquery.com/jquery-2.1.3.min.js',
```

Comment:
You shouldn't have to include jQuery right?

Reply:
The pagination plugin requires at least that version of jQuery, I assumed that we have an older version of jQuery

Reply:
I removed this line, and it seems to still work...

Reply:
Yes, i tried it too. Will remove it 👍